code/datums/station_traits/neutral_traits.dm

```diff
@@ -145,14 +145,67 @@
 	name = "Cargo Gorilla"
 	trait_type = STATION_TRAIT_NEUTRAL
 	weight = 1
+	sign_up_button = TRUE
 	show_in_report = FALSE // Selective attention test. Did you spot the gorilla?
-
+	force = TRUE // Laugh at me if this is still here in the PR
+	/// Who signed up to this in the lobby
+	var/list/lobby_candidates
 	/// The gorilla we created, we only hold this ref until the round starts.
 	var/mob/living/basic/gorilla/cargorilla/cargorilla
+	/// Have we handed out our gorilla pass yet?
+	var/accepting_signups = TRUE
 
 /datum/station_trait/cargorilla/New()
 	. = ..()
 	RegisterSignal(SSatoms, COMSIG_SUBSYSTEM_POST_INITIALIZE, PROC_REF(replace_cargo))
+	RegisterSignal(SSdcs, COMSIG_GLOB_PRE_GAMEMODE_SETUP, PROC_REF(on_gamemode_setup))
+
+/datum/station_trait/cargorilla/can_display_lobby_button()
+	return accepting_signups
+
+/datum/station_trait/cargorilla/setup_lobby_button(atom/movable/screen/lobby/button/sign_up/lobby_button)
+	RegisterSignal(lobby_button, COMSIG_ATOM_UPDATE_OVERLAYS, PROC_REF(on_lobby_button_update_overlays))
+	lobby_button.desc = "Become the Cargo Gorilla, a peaceful shepherd of boxes."
```

Comment:
```suggestion
	lobby_button.desc = "Sign up for Cargo Gorilla, a peaceful shepherd of boxes."
```
people arent guaranteed to get it

Reply:
They are if its latejoin

Reply:
now were at an impasse

Reply:
It shouldn't be on the lobby when the round has started, it should show on the normal job selection menu.

Reply:
OK I can remove it after the round has started (it did this initially but then I thought "but what if...")